TESTING.md

```diff
@@ -50,11 +52,18 @@ Say that you would want to debug a test in CI group 1, you can run the following
 
 This will print of an address, to which you could open your chrome browser on your instance and navigate to `chrome://inspect/#devices` and inspect the functional test runner `scripts/functional_tests.js`.
 
-### Additional checks
-Make sure you run lint checker before submitting a pull request. To run lint checker:
-`node scripts/precommit_hook.js --fix`
+### Backwards Compatibility tests
+To run all the backwards compatibility tests on vanilla dashboards:
 
-Please ensure that you don't introduce any broken links accidently. For any intentional broken link (e.g. dummy url in unit test), you can add it to [lycheeexclude](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.lycheeexclude) allow list specifically. 
```

Comment:
I think this got overwritten by a rebase.

Reply:
will change.